src/Schema/TableDiff.php

```diff
@@ -44,6 +44,17 @@ public function __construct(
         private readonly array $modifiedForeignKeys = [],
```

Comment:
Should this property be deprecated,  BTW?

Reply:
Is there a way to deprecate a promoted property? In https://github.com/doctrine/dbal/pull/6728, in order to deprecate `ForeignKeyConstraint#$options`, I had to replace a promoted constructor parameter into parameter and property.

But in this case, the property is private, so it's not covered by any BC promise by definition.

Reply:
Oh right, forget it then. And to answer your question, I thought it was possible to annotate them, but can't check right now.